brewfile.py

```diff
@@ -124,8 +126,13 @@ def _build_environs(self, data):
     # Handlers
 
     def _get_options(self, data):
-        stdout = data.get('stdout', self._default_stdout)
-        stderr = data.get('stderr', self._default_stderr)
+        defaults = self._context.defaults().get('brewfile', {})
```

Comment:
So, the only thing we do with this new `default` field (at least as far as I can tell) is changing how `stderr` and `stdout` are fetched.

But, we can already specify `stdout` and `stderr` fields inside your `- brewfile:` directive.

So, once again: I don't understand the reason behind `default`. Can you please explain in to me? You can use Russian if that's easier for you! 👍

Reply:
I want to set one configuration (debug or not) for a few Brewfiles because I don't want to change many configs when I debug the installation. 🙂

Reply:
Oh, I see. But this seems rather complicated. Are there any other solutions? For example, based on env vars?

Reply:
I reused native mechanism of dotbot. For example,[ Shell command used defaults too](https://github.com/anishathalye/dotbot/blob/0f5a5f33854c263fd15c03b20bbb44fe1e3398d6/dotbot/plugins/shell.py#L26).

Reply:
Do you think I should develop another way instead native? Russian is easier for me, but I try to practice my English. Sorry for my mistakes.

Reply:
Ok, I see. If this is the standard practice, then ok, let's merge it! 

I was not aware of this. Thanks for letting me know! 👍